src/main/java/uk/gov/companieshouse/psc/delta/config/ApplicationConfig.java

```diff
@@ -43,4 +57,18 @@ AvroSerializer serializer() {
     AvroDeserializer<ChsDelta> deserializer() {
         return new AvroDeserializer<>(ChsDelta.class);
     }
+
+    @Bean
+    public Supplier<InternalApiClient> internalApiClientSupplier() {
+        return () -> {
+            ApiKeyHttpClient apiKeyHttpClient = new ApiKeyHttpClient(apiKey);
+            apiKeyHttpClient.setRequestId(DataMapHolder.getRequestId());
+
+            InternalApiClient internalApiClient = new InternalApiClient(apiKeyHttpClient);
+            internalApiClient.setBasePath(apiUrl);
+            internalApiClient.setInternalBasePath(internalApiUrl);
```

Comment:
Internal api URL should not be required here and can be removed from app properties, config etc. Providing psc-data-api is the only API called using this client.